lib/trento/software_updates/discovery.ex

```diff
@@ -177,6 +177,16 @@ defmodule Trento.SoftwareUpdates.Discovery do
       |> Multi.run(:command_dispatching, fn _, _ ->
         dispatch_completion_command(host_id, discovered_health)
       end)
+      |> Multi.run(:notify_client, fn _, _ ->
+        with :ok <-
+               TrentoWeb.Endpoint.broadcast(
```

Comment:
The appropriate place to do this would be when a `SoftwareUpdatesHealthChanged` (possibly happening when a software updates discovery compeltes) event is emitted from the aggregate (I am hating myself for not having kept `SoftwareUpdatesDiscoveryCompleted` as a name... anyway) so I would move this broadcasting in `SoftwareUpdatesDiscoveryEventHandler`.
Similarly to what we do when broadcasting in projectors.